tools/osv-linter/internal/checks/affected.go

```diff
@@ -0,0 +1,21 @@
+package checks
+
+import (
+	"github.com/tidwall/gjson"
+)
+
+var CheckAffectedFieldValid = &CheckDef{
+	Code:        "A0001",
+	Name:        "affected-field-valid",
```

Comment:
This names it more like the existing `introduced-event-exists` check"

```suggestion
	Name:        "affected-data-exists",
```